### DISCUSSION TITLE
Welcome to mixin Discussions!

### DISCUSSION THREAD

**Discussion by cedricfung** [MixinNetwork/mixin#67] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by LazzaroneS** [comment]
I was always fighting for liberty recent years, whereas I can feel freedom in this community, I like here.

**Comment by cedricfung** [comment]
Organization discussions enabled, let's see how it works.
